src/python/pants/core/util_rules/subprocess_environment.py

```diff
@@ -22,11 +22,13 @@
     "ftp_proxy",
     "all_proxy",
     "no_proxy",
+    "requests_ca_bundle",
     "HTTP_PROXY",
     "HTTPS_PROXY",
     "FTP_PROXY",
     "ALL_PROXY",
     "NO_PROXY",
+    "REQUESTS_CA_BUNDLE",
```

Comment:
Perhaps offset this new block with a comment pointing to requests docs: https://requests.readthedocs.io/en/master/user/advanced/#ssl-cert-verification